src/test/java/com/synopsys/integration/alert/component/settings/PasswordResetServiceTest.java

```diff
@@ -110,7 +110,7 @@ public void resetPasswordValidTestIT() throws AlertException {
         addConfigurationFieldToMap(keyToFieldMap, EmailPropertyKeys.JAVAMAIL_PORT_KEY.getPropertyKey(), testProperties.getProperty(TestPropertyKey.TEST_EMAIL_SMTP_PORT));
 
         String username = "username";
-        UserModel userModel = UserModel.newUser(username, "", "noreply@synopsys.com", AuthenticationType.DATABASE, Set.of(), true);
+        UserModel userModel = UserModel.newUser(username, "", "noreply@blackducksoftware.com", AuthenticationType.DATABASE, Set.of(), true);
```

Comment:
This seems odd. Why did this get changed to blackducksoftware?

Reply:
the mail server we use for testing cannot resolve noreply@synopsys.com but it can resolve noreply@blackducksoftware.com since that is the default set in black duck for the sysadmin user.